README.md

```diff
@@ -1,7 +1,27 @@
 klog
 ====
 
-klog is a permanant fork of https://github.com/golang/glog. original README from glog is below
+klog is a permanant fork of https://github.com/golang/glog.
```

Comment:
`permanent`

Reply:
Going to fix, but interestingly, that wasn't my typo :wink: